### PR TITLE
Adding GCCL header for HTTP APIs.

### LINKED ISSUE #3006: Add more useful header info on HTTP requests
Most relevant place: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/9619004959def22ad2084691f382dc1cada46afd/core/google/cloud/_http.py#L177

May want to make this easier to use upstream

/cc @lukesneeringer

### PR DESCRIPTION
**Pull request by dhermes**
Continuation of #3035. Fixes #3006.

For now just a single commit: "Adding `__version__` in packages that need HTTP." because I need to step AFK for a bit.

### CODE REVIEW

File: monitoring/google/cloud/monitoring/__init__.py [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 """Google Stackdriver Monitoring API wrapper."""
 
+from pkg_resources import get_distribution
```
Comment (daspecster) [inline monitoring/google/cloud/monitoring/__init__.py]: Two newlines above this to match the others?

File: bigquery/google/cloud/bigquery/__init__.py [hunk 2]
```diff
@@ -23,6 +23,9 @@
 """
 
 
+from pkg_resources import get_distribution
+__version__ = get_distribution('google-cloud-bigquery')
```
Comment (daspecster) [inline bigquery/google/cloud/bigquery/__init__.py]: I feel like the package name should be available as a constant somewhere already right?

File: runtimeconfig/google/cloud/runtimeconfig/__init__.py [hunk 3]
```diff
@@ -14,4 +14,7 @@
 
 """Google Cloud Runtime Configurator API package."""
 
+from pkg_resources import get_distribution
```
Comment (daspecster) [inline runtimeconfig/google/cloud/runtimeconfig/__init__.py]: Ditto two newlines.

File: speech/google/cloud/speech/__init__.py [hunk 4]
```diff
@@ -14,6 +14,9 @@
 
 """Google Cloud Speech API wrapper."""
```
Comment (daspecster) [inline speech/google/cloud/speech/__init__.py]: ditto two new lines.

File: speech/google/cloud/speech/__init__.py [hunk 5]
```diff
@@ -14,6 +14,9 @@
...
 
+from pkg_resources import get_distribution
+__version__ = get_distribution('google-cloud-speech')
+
```
Comment (daspecster) [inline speech/google/cloud/speech/__init__.py]: ditto two newlines.

File: translate/google/cloud/translate/__init__.py [hunk 6]
```diff
@@ -14,6 +14,9 @@
 
 """Google Cloud Translation API wrapper."""
 
+from pkg_resources import get_distribution
```
Comment (daspecster) [inline translate/google/cloud/translate/__init__.py]: ditto two newlines.

File: vision/google/cloud/vision/__init__.py [hunk 7]
```diff
@@ -14,4 +14,7 @@
 
 """Google Cloud Vision API package."""
```
Comment (daspecster) [inline vision/google/cloud/vision/__init__.py]: ditto two newlines.

File: bigquery/google/cloud/bigquery/__init__.py (hunk 2 above)
Comment (lukesneeringer) [inline bigquery/google/cloud/bigquery/__init__.py]: Not that I am aware of.

**Comment by dhermes** [comment]
OK finally got time back at the keyboard.

@lukesneeringer @daspecster PTAL

**Pull request merged**